ras_party/controllers/account_controller.py

```diff
@@ -331,6 +331,14 @@ def change_respondent_account_status(payload, party_id, session):
     if not respondent:
         raise ClientError("Respondent does not exist", respondent_id=party_id,
                           status=404)
+    # Unlock respondents account
+    if status == 'ACTIVE':
+        email_address = respondent.email_address
+        oauth_response = OauthClient().update_account(username=email_address, account_locked='False')
+
+        if oauth_response.status_code != 201:
```

Comment:
what about use raise_for_status ?

Reply:
Do youhave any unit test for this code ?

Reply:
i did think `raise_for_status` would be best practice, but i followed the pattern of the file. And it is tested through another integration style test

Reply:
Sorry but there is not such thing as pattern of a file, you should chose teh best option or practice regardless what other developers more or lesse experinced do . Not sure that test_put_change_respondent_account_status_active is actually testing something. Plus you added a new condition checking status code != 201 so u should test you raise RasError.

Reply:
ok sure ill use `raise_for_status` and will double check the tests